sdk/src/compressed_account.rs

```diff
@@ -502,6 +233,32 @@ pub struct CompressedAccountWithMerkleContext {
     pub merkle_context: MerkleContext,
 }
 
+impl CompressedAccountWithMerkleContext {
+    pub fn new_init_account(
```

Comment:
`new_init_account` sounds weird to me.
What about `new` , `init_account` or `init_new_account` or `init` ?

Reply:
Actually I think I can remove this block all together, it's a leftover from before I introduced `LightAccountMeta`/`LightAccountInfo`.